src/utils/generateJailsChartData.js

```diff
@@ -0,0 +1,127 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2020 Recidiviz, Inc.
```

Comment:
Whoops: new license text should be 2021 now.

Reply:
Should I fix this for all files in this PR?

Reply:
Yes please! And in future PRs, too, but don't worry about files that predate your PR.